src/main/java/core/basesyntax/dao/PurchaseOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.FruitDB;
+
+public class PurchaseOperation implements FruitOperation {
```

Comment:
such naming makes confusion with model Operation
please make interface OperationHandler instead of FruitOperation and implementations PurchaseOperationHandler and so on